src/Widgets/Tabs/Geometry.hs

```diff
@@ -43,19 +43,28 @@ import qualified SmallGL
 
 
 panelGeometry :: Reflex t
-              => SmallGL.RenderingApi
+              => Dynamic t Bool
```

Comment:
Please, add comments to such arguments using "`-- ^ `" style; otherwise it is too easy to mix which Bool stands for what.